polaris/polaris/sep6/urls.py

```diff
@@ -0,0 +1,16 @@
+from django.urls import path
+from django.conf import settings
+from polaris.sep6 import info, deposit, withdraw, fee, transaction
+
+urlpatterns = [
+    path("deposit", deposit.deposit),
+    path("withdraw", withdraw.withdraw),
+    path("info", info.info),
+    path("fee", fee.fee),
+    path("transaction", transaction.transaction),
+    path("transactions", transaction.transactions),
+]
+if "sass_processor" in settings.INSTALLED_APPS:
```

Comment:
whats the meaning of sass_processor as app logic?

Reply:
Explained in the related comment. UI pages require `sass_processor` but UI pages (more_info_url) are optional in SEP-6.

Reply:
so we're using the existence of the sass processor as an indicator that there is a more_info_url?  Can we do something more explicit? If we stop using sass processor then this logic will break

Reply:
The requirement of having `sass_processor` installed will be removed before the next release. When that happens we can remove these conditions.